cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -102,6 +104,26 @@ pNetworkId envCli = asum $ mconcat
     pure <$> maybeToList (envCliNetworkId envCli)
   ]
 
+pTarget :: CardanoEra era -> Parser (Consensus.Target ChainPoint)
```

Comment:
So have a look at how `pQueryDRepStateCmd` is implemented. You should parameterize this parser on `ConwayEraOnwards era`. The problem here is when we have a new era, we will have to remember to update this parser. GHC will not warn us because of the wildcard pattern  `_`.

Reply:
Thanks, I was just wondering about that when I wrote the `_`